### PR TITLE
Recognize JPG type to display in download list

### PR DESCRIPTION
**Pull request by tkohr**
PR recognizes JPG file type in links with protocol `WWW:DOWNLOAD` and displays it in the download list, which is currently not the case ("unknown"). To test this behavior, search for entry "Schémas de cohérence territoriale dans l'Oise" (471011ac-b927-4c8d-9589-ce3c530ea08d) for example.

### CODE REVIEW

**Comment by fgravin** [comment]
Thanks @tkohr, please add a description, with an exemple of a failing link (you can take the one i wrote in Jira).
Also, you must add a test which would break without your fix and which now passes.

Thanks

**Comment by tkohr** [comment]
Yes right @fgravin, I added the missing tests for the file types.

File: libs/feature/record/src/lib/links/link-utils.spec.ts [hunk 1]
```diff
@@ -4,24 +4,86 @@ import {
   checkFileFormat,
   getLinksWithEsriRestFormats,
 } from './link-utils'
+import { LINK_FIXTURES } from './link.fixtures'
 
 describe('link utils', () => {
   describe('#getDownloadFormat', () => {
-    describe('for a FILE link', () => {
+    describe('for a csv FILE link', () => {
       it('returns download format', () => {
```
Comment (fgravin) [inline libs/feature/record/src/lib/links/link-utils.spec.ts:12]:
```suggestion
      it('returns csv format', () => {
```

File: libs/feature/record/src/lib/links/link.fixtures.ts [hunk 2]
```diff
@@ -0,0 +1,108 @@
+export const LINK_FIXTURES = {
```
Comment (fgravin) [inline libs/feature/record/src/lib/links/link.fixtures.ts:1]: good !

**Review by fgravin — CHANGES_REQUESTED** [review]
minor comment but it looks good to me thanks.
You'll have to wait for the big refactoring PR to be merged and then update your PR accordingly, sorry for that.

**Comment by tkohr** [comment]
You are right @fgravin, the description wasn't quite clear. Updated that, rebased and integrated your suggestion. If CI ends up green and you are ok with it, ready to merge I think.

**Review by fgravin — APPROVED** [review]

**Pull request merged**